### PR TITLE
Openphishv2

### PR DESCRIPTION
**Pull request by JasBeilin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28227

## Description
A new version for OpenPhish integration based on Python instead of JS.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+''' IMPORTS '''
+
+import requests
+import traceback
+from typing import Dict
+
+requests.packages.urllib3.disable_warnings()
+
+# Disable insecure warnings
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:12]:
```suggestion
# Disable insecure warnings
requests.packages.urllib3.disable_warnings()
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        - contextPath: URL.Malicious.Description
+          description: For malicious URLs, the reason for the vendor to make the decision
+        - contextPath: DBotScore.Indicator
+          description: The indicator we tested
+        - contextPath: DBotScore.Type
+          description: The type of the indicator
+        - contextPath: DBotScore.Vendor
+          description: Vendor used to calculate the score
+        - contextPath: DBotScore.Score
+          description: The actual score
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml:50]: https://xsoar.pan.dev/docs/integrations/context-standards#url
https://xsoar.pan.dev/docs/integrations/context-standards#dbot-score
I'm not sure this follows the standards, please review all outputs.
```suggestion
      outputs:
        - contextPath: URL.Data
          description: The URL.
        - contextPath: URL.Malicious.Vendor
          description: The vendor reporting the URL as malicious.
        - contextPath: URL.Malicious.Description
          description: For malicious URLs, the reason for the vendor to make the decision
        - contextPath: DBotScore.Indicator
          description: The indicator we tested
        - contextPath: DBotScore.Type
          description: The type of the indicator
        - contextPath: DBotScore.Vendor
          description: The vendor used to calculate the score.
        - contextPath: DBotScore.Score
          description: The actual score
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_description.md [hunk 3]
```diff
@@ -0,0 +1 @@
+In order to configure an instance you can configure interval from fetching data from the api.
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_description.md:1]: @kirbles19 We would love your help here. to configure this integration you do not need creds. just add the refresh interval. all the urls are stored in the integration context and are being refreshed every x time.

File: Packs/OpenPhish/.secrets-ignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+http://www.niccakorea.com/board/index.html
+http://lloyds.settlemypayee.uk
+https://whatsapp-chat02.zzux.com
+http://dd0ddddddcuser.ey.r.appspot.com
+http://goo.co
```
Comment (bakatzir) [inline Packs/OpenPhish/.secrets-ignore:5]: if it is only for test, maybe we should sanitize them?
```suggestion
hxxp://www.niccakorea.com/board/index.html
hxxp://lloyds.settlemypayee.uk
hxxps://whatsapp-chat02.zzux.com
hxxp://dd0ddddddcuser.ey.r.appspot.com
hxxp://goo.co
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+import pytest
+import OpenPhish_v2
+import demistomock as demisto
+from OpenPhish_v2 import Client, _is_reload_needed, remove_backslash, reload_command, status_command, url_command
+from freezegun import freeze_time
+from test_data.api_raw import RAW_DATA
+MOCK_URL = "http://openphish.com"
+MOCK_DELIVERED_MESSAGE = {}
+
+RELOADED_DATA = [
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py:11]: let's sanitize the urls

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+               'http://dd0ddddddcuser.ey.r.appspot.com'],
+      "timestamp": "1601532000000"},
+     {'URL': [{'Data': 'http://goo.co'}],
+      'DBOTSCORE': [{'Indicator': 'http://goo.co', 'Type': 'url', 'Vendor': 'OpenPhish', 'Score': 0}]}
+     )
+]
+
+
+@pytest.mark.parametrize('url,context,expected_results', CONTEXT_MOCK_WITH_URL)
+def test_url_command(mocker, url, context, expected_results):
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py:163]: please add given: a url, when: "mocking the integration context data, running url_command", then: .... validating...

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+        - Returns number of loaded urls if data was loaded.
+        - Otherwise, returns Database not loaded.
+    """
+    client = Client(MOCK_URL, True, False, 1)
+    mocker.patch.object(demisto, "getIntegrationContext", return_value=data)
+    status = status_command(client)
+    assert status.readable_output == expected_result
+
+
+CONTEXT_MOCK_WITH_URL = [
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_test.py:139]: please sanitize

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
+OpenPhish uses proprietary Artificial Intelligence algorithms to automatically identify zero-day phishing sites and provide comprehensive, actionable, real-time threat intelligence.
+
+## Configure OpenPhish_v2 on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for OpenPhish_v2.
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:6]:
```suggestion
2. Search for OpenPhish v2.
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+            "Score": 0,
+            "Type": "url",
+            "Vendor": "OpenPhish"
+        },
+        {
+            "Indicator": "http://dd0ddddddcuser.ey.r.appspot.com/",
+            "Score": 0,
+            "Type": "url",
+            "Vendor": "OpenPhish"
+        }
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:65]: is not this supposed to be malicious? please do one that is, and sanitize

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+There is no context output for this command.
+
+#### Command Example
+```!openphish-reload```
+
+#### Context Example
+```
+{}
+```
+
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:110]: rm

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+There is no context output for this command.
+
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:102]: rm

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+`openphish-status`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+There is no context output for this command.
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:131]: rm

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+#### Command Example
+```!openphish-status```
+
+#### Context Example
+```
+{}
+```
+
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:140]: rm

File: Packs/OpenPhish/Integrations/OpenPhish_v2/command_examples.txt [hunk 14]
```diff
@@ -0,0 +1,3 @@
+!url using-brand=OpenPhish_v2 url="google.com, http://dd0ddddddcuser.ey.r.appspot.com/"
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/command_examples.txt:1]:
```suggestion
!url using-brand=OpenPhish_v2 url="google.com, hxxp://dd0ddddddcuser.ey.r.appspot.com/"
```

File: Packs/OpenPhish/ReleaseNotes/1_1_0.md [hunk 15]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: OpenPhish v2
+- OpenPhish uses proprietary Artificial Intelligence algorithms to automatically identify zero-day phishing sites and provide comprehensive, actionable, real-time threat intelligence.
```
Comment (bakatzir) [inline Packs/OpenPhish/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added a v2 for the OpenPhish integration that provides enrichment for URLs.
```

Comment (bakatzir) [reply Packs/OpenPhish/ReleaseNotes/1_1_0.md:4]: @kirbles19 your inputs

File: Packs/OpenPhish/pack_metadata.json [hunk 16]
```diff
@@ -1,16 +1,16 @@
...
+    "currentVersion": "1.1.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-04-14T00:00:00Z",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [],
+    "useCases": [],
```
Comment (bakatzir) [inline Packs/OpenPhish/pack_metadata.json:14]:
```suggestion
    "useCases": ["Phishing"],
```

File: Packs/OpenPhish/pack_metadata.json [hunk 17]
```diff
@@ -1,16 +1,16 @@
...
-  "categories": [
-    "Data Enrichment & Threat Intelligence"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
+    "name": "OpenPhish",
+    "description": "OpenPhish uses proprietary Artificial Intelligence algorithms to automatically identify zero-day phishing sites and provide comprehensive, actionable, real-time threat intelligence.",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
```
Comment (bakatzir) [inline Packs/OpenPhish/pack_metadata.json:5]:
```suggestion
    "currentVersion": "2.0.0",
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 18]
```diff
@@ -0,0 +1,223 @@
...
+
+demisto.debug(f'Command being called is {demisto.command()}')
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    # get the service API url
+    base_url = API_PATH
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:185]: we are using the global var API_PATH once. I think stating the url here, or in the client will be nicer, and no need for the global var.

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 19]
```diff
@@ -0,0 +1,223 @@
...
+
+    commands = {
+        'url': url_command,
+        'openphish-reload': reload_command,
+        'openphish-status': status_command,
+    }
+
+    hours_to_refresh = demisto.params().get('fetchIntervalHours', '1')
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:193]:
```suggestion
    params = demisto.params()
    hours_to_refresh = params.get('fetchIntervalHours', '1')
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 20]
```diff
@@ -0,0 +1,223 @@
...
+
+    try:
+        hours_to_refresh = float(hours_to_refresh)
+        use_ssl = not demisto.params().get('insecure', False)
+        use_proxy = demisto.params().get('proxy', False)
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:198]:
```suggestion
        use_ssl = not params.get('insecure', False)
        use_proxy = params.get('proxy', False)
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 21]
```diff
@@ -0,0 +1,223 @@
...
+
+        command = demisto.command()
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
+
+        elif command in commands:
+            return_results(commands[command](client, **demisto.args()))
+
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:213]: no need for newline in between if/elif
```suggestion
        if command == 'test-module':
            result = test_module(client)
            return_results(result)
        elif command in commands:
            return_results(commands[command](client, **demisto.args()))
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 22]
```diff
@@ -0,0 +1,223 @@
...
+        md += "Last load time **" + timestamp_to_datestring(data.get('timestamp'),
+                                                            "%a %b %d %Y %H:%M:%S (UTC)",
+                                                            is_utc=True) + "**\n"
+    else:
+        md += "Database not loaded.\n"
+
+    return CommandResults(readable_output=md)
+
+
+demisto.debug(f'Command being called is {demisto.command()}')
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:177]: let's put it in main

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 23]
```diff
@@ -0,0 +1,223 @@
...
+        data = client.http_request('feed.txt', resp_type='text')
+        data = data.splitlines()
+        data = list(map(remove_backslash, data))
+
+        context = {"list": data,
+                   "timestamp": date_to_timestamp(datetime.now(), DATE_FORMAT)}
+        set_integration_context(context)
+
+    except NotFoundError as e:
+        return_error('Check server URL - ' + e.message)
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:74]: let's use raise here, and f notation

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 24]
```diff
@@ -0,0 +1,223 @@
...
+
+
+def _save_urls_to_instance(client: Client):
+    """
+    gets urls from api and load it to instance's memory
+    raise NotFoundError if the http request to the api failed.
+
+    """
+    try:
+        # gets the urls from api with unite format
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:64]:
```suggestion
        # gets the urls from api and formats them
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 25]
```diff
@@ -0,0 +1,223 @@
...
+
+def url_command(client: Client, **kwargs) -> CommandResults:
+    data = get_integration_context()
+    if _is_reload_needed(client, data):
+        reload_command(client)
+        data = get_integration_context()
+
+    url_object_list = []
+    if not data:
+        raise DemistoException("Data was not saved correctly")
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:135]:
```suggestion
        raise DemistoException("Data was not saved correctly to the integration context.")
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 26]
```diff
@@ -0,0 +1,223 @@
...
+
+    url_list_from_user = argToList(kwargs.get('url'))
+    markdown = "### OpenPhish Database - URL Query\n"
+    urls_in_db = data.get('list', [])
+    for url in url_list_from_user:
+        url_fixed = remove_backslash(url)
+        if url_fixed in urls_in_db:
+            dbotscore = Common.DBotScore.BAD
+            desc = 'Match found in OpenPhish database'
+            markdown += "#### Found matches for given URL " + url + "\n"
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:145]: please use f notation

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 27]
```diff
@@ -0,0 +1,223 @@
...
+        else:
+            dbotscore = Common.DBotScore.NONE
+            desc = ""
+            markdown += "#### No matches for URL " + url + "\n"
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:149]: please use f notation

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 28]
```diff
@@ -0,0 +1,223 @@
...
+
+        dbot = Common.DBotScore(url, DBotScoreType.URL, 'OpenPhish', dbotscore, desc)
+        url_object_list.append(Common.URL(url, dbot))
+
+    return CommandResults(indicators=url_object_list, readable_output=markdown)
+
+
+def reload_command(client: Client, **kwargs) -> CommandResults:
+    _save_urls_to_instance(client)
+    return CommandResults(readable_output='updated successfully')
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:159]:
```suggestion
    return CommandResults(readable_output='Database was updated successfully to the integration context.')
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 29]
```diff
@@ -0,0 +1,223 @@
...
+    data = get_integration_context()
+
+    md = "OpenPhish Database Status\n"
+    if data and data.get('list', None):
+        md += "Total **" + str(len(data.get('list'))) + "** URLs loaded.\n"
+        md += "Last load time **" + timestamp_to_datestring(data.get('timestamp'),
+                                                            "%a %b %d %Y %H:%M:%S (UTC)",
+                                                            is_utc=True) + "**\n"
+    else:
+        md += "Database not loaded.\n"
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:172]: please use f notation

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 30]
```diff
@@ -0,0 +1,223 @@
...
+    """
+
+    def __init__(self, message):
+        self.message = message
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:38]: copy pasta?

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 31]
```diff
@@ -0,0 +1,223 @@
...
+    """
+
+    def __init__(self, url: str, use_ssl: bool, use_proxy: bool, fetch_interval_hours: float = 1):
+        super().__init__(url, verify=use_ssl, proxy=use_proxy)
+        self.fetch_interval_hours = fetch_interval_hours
+
+    def http_request(self, name, resp_type=None):
+        """
+        initiates a http request to a test url
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:47]:
```suggestion
        initiates a http request to openphish
```

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py [hunk 32]
```diff
@@ -0,0 +1,223 @@
...
+        return True
+    now = datetime.now()
+
+    if data.get('timestamp') <= date_to_timestamp(now - timedelta(hours=client.fetch_interval_hours)):
+        return True
+
+    return False
+
+
+def test_module(client) -> str:
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py:97]:
```suggestion
def test_module(client: Client) -> str:
```

**Review by bakatzir — COMMENTED** [review]
Very good and comprehensive first PR.
Loved the extensive UT.
please note the f notation in the code, and removing non used sections in the README.

**Review by bakatzir — COMMENTED** [review]
Also let's deprecate the old integration

File: Packs/OpenPhish/Integrations/OpenPhish_v2/README.md [hunk 33]
```diff
@@ -0,0 +1,143 @@
+OpenPhish uses proprietary Artificial Intelligence algorithms to automatically identify zero-day phishing sites and provide comprehensive, actionable, real-time threat intelligence.
+
+## Configure OpenPhish_v2 on Cortex XSOAR
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/README.md:3]:
```suggestion
## Configure OpenPhish v2 on Cortex XSOAR
```

**Comment by bakatzir** [comment]
@kirbles19 please review: 

[link](https://github.com/demisto/content/blob/00e67ed4e4b7d51c907d6c075c0d1f2e6cffa1dd/Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml)

File: Packs/OpenPhish/ReleaseNotes/1_1_0.md (hunk 15 above)
Comment (kirbles19) [reply Packs/OpenPhish/ReleaseNotes/1_1_0.md:4]: Created v2 of the OpenPhish integration, which now provides URL enrichment.

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_description.md (hunk 3 above)
Comment (kirbles19) [reply Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2_description.md:1]: To configure an integration instance, you only need to set the refresh interval. All of the URLs are stored in the integration context and are refreshed according to the refresh interval.

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@JasBeilin Done

**Review by bakatzir — APPROVED** [review]
please fix the configuration instructions in the readme as well.

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml [hunk 34]
```diff
@@ -0,0 +1,64 @@
...
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+description: OpenPhish uses proprietary Artificial Intelligence algorithms to automatically
+  identify zero-day phishing sites and provide comprehensive, actionable, real-time
+  threat intelligence.
+name: OpenPhish_v2
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml:24]: New feature(integrations)s are only from 5.0
```suggestion
fromversion: 5.0.0
name: OpenPhish_v2
```

Comment (bakatzir) [reply Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml:24]: my bad

File: Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml [hunk 35]
```diff
@@ -0,0 +1,64 @@
...
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Database refresh interval (hours)
+  name: fetchIntervalHours
+  defaultvalue: 1
+  type: 0
+  required: false
```
Comment (bakatzir) [inline Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml:20]:
```suggestion
- display: Database refresh interval (hours)
  name: fetchIntervalHours
  defaultvalue: 1
  type: 0
  required: false
- display: Use system proxy settings
  name: proxy
  defaultvalue: ""
  type: 8
  required: false
- display: Trust any certificate (not secure)
  name: insecure
  defaultvalue: ""
  type: 8
  required: false
```

**Pull request merged**